pyproject.toml

```diff
@@ -49,6 +49,23 @@ tinydb = "^4.5.2"
 munch = "^2.5.0"
 hupper = "^1.10.3"
 qualang-tools = "^0.9.0"
+hiredis = "^2.0.0"
+redis = "^4.1.0"
+Jinja2 = "^3.0.3"
+psutil = "^5.9.0"
+pyzmq = "^22.3.0"
+msgpack = "^1.0.3"
+pika = "^1.2.0"
+SQLAlchemy = "^1.4.32"
```

Comment:
what is the difference between this and line 32?

Reply:
I removed now line 59; it was added automatically by poetry, strange that it didn't do it in case insensitive way.